homeassistant/components/goalzero/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Goal Zero Yeti",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Goal Zero Yeti",
+        "data": {
+          "host": "host",
+          "name": "name"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "cannot connect",
+      "connect_timeout": "connect timeout",
+      "host_required": "host required",
+      "invalid_host": "invalid host",
+      "unknown": "unknown error"
```

Comment:
These values should be more descriptive as they are shown to the user. So make them either match an existing placeholder or a proper sentence (that can be translated as well).